Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewController.swift

```diff
@@ -259,6 +259,13 @@ final class NoShippingPledgeViewController: UIViewController,
         self?.pledgeRewardsSummaryViewController
           .configureWith(rewardsData: rewardsData, bonusAmount: bonusAmount, pledgeData: pledgeData)
       }
+    
+    self.viewModel.outputs.showPledgeOverTimeUI
+          .observeForUI()
+          .observeValues { value in
+            // TODO: Use this flag to hide or show the Pledge Over Time UI [MBL-1814] https://kickstarter.atlassian.net/browse/MBL-1814
```

Comment:
```suggestion
            // TODO: Use this flag to hide or show the Pledge Over Time UI [MBL-1814](https://kickstarter.atlassian.net/browse/MBL-1814)
```

nit: adding parenthesis around the url will make it clickable from Xcode for quick access